src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -15080,6 +15080,20 @@ await VerifyExpectedItemsAsync(markup, [
         ]);
     }
 
+    [Fact]
+    public async Task PartialPropertyOrConstructor()
+    {
+        var markup = """
+            partial class C
+            {
+                partial $$
```

Comment:
can you add as part of the theory, an additional 'modifiers' section, where you do things like `static` and `private`?